src/envvars.sh

```diff
@@ -0,0 +1,2 @@
+export REDIRECT_PORT=$((KOLIBRI_HTTP_PORT+1))
```

Comment:
If KOLIBRI_HTTP_PORT is unset, this is going to set REDIRECT_PORT to 1. That's likely the source of the _permission denied_ error people are getting :)

Could do something like…

    export REDIRECT_PORT="${REDIRECT_PORT:-8080}"

Reply:
Thanks! That looks to be the culprit. In the previous round of dev, I must have had that set externally from some earlier testing. I was able to replicate the error locally, and your suggestion fixes it.